src/logger/types.ts

```diff
@@ -49,7 +49,7 @@ export class Logger {
     const time = moment().format(timeFormat)
 
     const timeText = clc.blackBright(time)
-    const titleText = clc.bold(this.colors[level](this.scope))
+    const titleText = clc.bold(this.colors[level](`[Messaging] ${this.scope}`))
```

Comment:
Having `[Messaging]` in front of every log is annoying if messaging is started seperatly. So it would be better to only have that if messaging was spinned from the main repo.